docs/index.md

```diff
@@ -19,9 +19,10 @@ Scarf provides open-source software maintainers with deep insights about their p
 9. **[Open Source Qualified Leads (OQLs)](/oql):** Identify organizations and individuals that have engaged with your OSS.
 10. **[Open Source Adoption Funnel Stages](/funnel-stages):** Understand the level of engagement of potential leads.
 11. **[Data Export](/data-export):** Export your data for customized analysis. Explore the potential of data manipulation to suit your needs.
-12. **[User Guide & Best Practices](/user_best_practices):** Make a plan to use the data uncovered in your OQLs.
-13. **[Troubleshooting](/troubleshooting):** Find solutions to common issues and questions.
-14. **[API Docs](https://api-docs.scarf.sh/v2.html):** Understand the capabilities of a specific API version and its available endpoints.
+12. **[Event Import](/event-import):** Import your data
+13. **[User Guide & Best Practices](/user_best_practices):** Make a plan to use the data uncovered in your OQLs.
+14. **[Troubleshooting](/troubleshooting):** Find solutions to common issues and questions.
+16. **[API Docs](https://api-docs.scarf.sh/v2.html):** Understand the capabilities of a specific API version and its available endpoints.
```

Comment:
```suggestion
15. **[API Docs](https://api-docs.scarf.sh/v2.html):** Understand the capabilities of a specific API version and its available endpoints.
```

Reply:
argh. i thought i corrected this. thank you! :pray: